platform/web/export/export_plugin.cpp

```diff
@@ -345,6 +347,7 @@ void EditorExportPlatformWeb::get_export_options(List<ExportOption> *r_options)
 	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "html/focus_canvas_on_start"), true));
 	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "html/experimental_virtual_keyboard"), false));
 	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "progressive_web_app/enabled"), false));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "progressive_web_app/ensure_cross-origin_isolation_headers"), true));
```

Comment:
Do we want `-` in a property name? Is it used anywhere else? It feels like it breaks the style generally

Reply:
"cross-origin" is a word by itself. It does not fit with the _ as a space rule.
https://web.dev/articles/coop-coep

Reply:
I think it would still be less surprising if it's properly snake_case, where `-` would be invalid.

Reply:
I renamed to `progressive_web_app/ensure_cross_origin_isolation_headers`

Reply:
Oh, though. The generated label text for the option will have it's hyphen missing.